steam_wrapper/steam_wrapper.py

```diff
@@ -209,29 +202,45 @@ def apply(self):
         return False
 
 
-def migrate_config():
+
+def _get_host_xdg_mounts(xdg_name: str, flatpak_info):
+    assert xdg_name in ["xdg-data", "xdg-config", "xdg-cache"]
+    dirs: t.Set[str] = set()
+    for filesystem in flatpak_info["filesystems"]:
+        filesystem_path = filesystem.rsplit(":", 1)[0]
+        path_seq = os.path.split(filesystem_path)
```

Comment:
`os.path.split()` is misused here. Will work for first-level subdirs, break otherwise. Sorry for bad suggestion. Here should be something like
```suggestion
        path_seq = os.path.normpath(filesystem_path).split(os.path.sep)
```